src/view/TextAreaView.java

```diff
@@ -46,12 +56,46 @@ public TextAreaView(TextAreaViewModel textAreaViewModel) {
 
         // Add the mainPanel to the CENTER of TextAreaView
         this.add(mainPanel, BorderLayout.CENTER);
+
+        clearButton.addActionListener(new ActionListener() {
+            final ClearViewModel clearViewModel = new ClearViewModel();
+
+            final ClearOutputBoundary clearOutputBoundary = new ClearPresenter(clearViewModel, textArea);
+
+            final ClearInputBoundary clearUseCaseInteractor = new ClearInteractor(clearOutputBoundary);
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                if (e.getSource().equals(clearButton)) {
+
+                    ClearController clearController = new ClearController(clearUseCaseInteractor);
+                    clearController.execute();
+                }
+            }
+        });
+
+        saveButton.addActionListener(new ActionListener() {
+
+            final SaveViewModel saveViewModel = new SaveViewModel();
+
+            final SaveOutputBoundary saveOutputBoundary = new SavePresenter(saveViewModel);
+
+            final SaveInputBoundary saveUseCaseInteractor = new SaveInteractor(saveOutputBoundary);
```

Comment:
SavePresenter and SaveInteractor cannot be directly created in View as it violates CA.